src/song_utils.py

```diff
@@ -8,11 +9,55 @@
 from mutagen.id3 import ID3FileType, PictureType
 from mutagen.ogg import OggFileType
 from mutagen.wave import WAVE
-from nextcord import File
+from nextcord import File, Embed, Message, Attachment, User, Member
+from nextcord.utils import escape_markdown
 from PIL import Image, UnidentifiedImageError
 
 import config
 
+def embed_song(submit_message: Message | str, attachment_filepath: str, attachment: Attachment, user: User | Member, random_emoji=None) -> Embed:
```

Comment:
I think it would be cleaner to call "random_emoji" just "emoji" and make it nonoptional, forcing the caller to provide the random one if they are missing it